lib/docker/template/cache.rb

```diff
@@ -45,9 +43,10 @@ def aliased_context(builder)
       # ----------------------------------------------------------------------
 
       def cleanup(repo)
+        return unless repo.clean_cache?
```

Comment:
Well, I don't know if you are referring to the cleanup behaviour change or the use of `return unless` specifically.

About the first, as a pointed out in my pull request, you have method clean, and --clean --no-clean directives in your CLI API. But they were meaningless. You were always cleaning the cache. I needed specifically the cache. So I fixed what as an user I would expect from CLI commands and help descriptions. This code specifically skips cleanup if is not passed by the user as listed in the pull request and commit messages.

About the instruction itself, I must say that I'm not a Ruby developer. I have little to none experience and I studied your code to try following your style. My apologies, I was made with best intention and must care.